crates/tauri/src/lib.rs

```diff
@@ -37,6 +37,7 @@
 //! - **macos-proxy**: Adds support for [`WebviewBuilder::proxy_url`] on macOS. Requires macOS 14+.
 //! - **specta**: Add support for [`specta::specta`](https://docs.rs/specta/%5E2.0.0-rc.9/specta/attr.specta.html) with Tauri arguments such as [`State`](crate::State), [`Window`](crate::Window) and [`AppHandle`](crate::AppHandle)
 //! - **dynamic-acl** *(enabled by default)*: Enables you to add ACLs at runtime, notably it enables the [`Manager::add_capability`] function.
+//! - **url-pattern**: Enables using the [URLPattern] spec for URL scopes on JavaScript APIs.
```

Comment:
We probably need the link for this to work

```suggestion
//! - **url-pattern**: Enables using the [`URLPattern`](https://developer.mozilla.org/en-US/docs/Web/API/URLPattern) spec for URL scopes on JavaScript APIs.
```